MAINTAINERS.md

```diff
@@ -1,6 +1,139 @@
+This document defines the roles available in the oneDPL project and provides a list of the
+[Current Code Owners](#currentcodeowners) and [Current Maintainers](#currentmaintainers) for each area of development
+in oneDPL. The document also defines the process for maintainers to [Leave A Role](#leavearole) in the project.
+
+# Roles and Responsibilities
+
+The oneDPL project defines three primary roles:
+* [Contributor](#contributor)
+* [Code Owner](#codeowner)
+* [Maintainer](#maintainer)
+
+[permissions]: https://docs.github.com/en/organizations/managing-user-access-to-your-organizations-repositories/managing-repository-roles/repository-roles-for-an-organization#permissions-for-each-role
+
+|                                                                                                                                             |       Contributor       |       Code Owner        |       Maintainer        |
+| :------------------------------------------------------------------------------------------------------------------------------------------ | :---------------------: | :---------------------: | :---------------------: |
+| _Responsibilities_                                                                                                                          |                         |                         |                         |
+| Follow the Code of Conduct                                                                                                                  |            ✓            |            ✓           |            ✓            |
+| Follow Contribution Guidelines                                                                                                              |            ✓            |            ✓           |            ✓            |
+| Enforce Contribution Guidelines                                                                                                             |            ✗            |            ✓           |            ✓            |
+| Co-own component or aspect of the library,<br>  including contributing: bug fixes, implementing features,<br> and performance optimizations |            ✗            |            ✓           |            ✓            |
+| Co-own on technical direction of component or<br> aspect of the library                                                                     |            ✗            |            ✗           |            ✓            |
+| Co-own the project as a whole,<br> including determining strategy and policy for the project                                                |            ✗            |            ✗           |            ✓            |
+| _Privileges_                                                                                                                                |                         |                         |                         |
```

Comment:
When looking at the generated table, it could be difficult to discern that this is a header row for a new section and everything that follows are privileges whereas everything before are distinct responsibilities. 

Would it make sense to split this table in two, one for responsibilities and one for privileges?

Reply:
I've made the split.  I think it does make it clearer, but the different table widths now is a bit annoying to look at in my opinion.

Reply:
I've reverted to the single table, but at the suggestion of others offline added a blank row and then repeated the roles as an internal header.  I think this makes it clear that the Privileges row is an internal header.